tensorflow/tensorflow.bzl

```diff
@@ -379,13 +385,43 @@ def _rpath_linkopts(name):
 
 # Bazel-generated shared objects which must be linked into TensorFlow binaries
 # to define symbols from //tensorflow/core:framework and //tensorflow/core:lib.
-def tf_binary_additional_srcs():
-    return if_static(
-        extra_deps = [],
-        otherwise = [
-            clean_dep("//tensorflow:libtensorflow_framework.so"),
+def tf_binary_additional_srcs(fullversion = False):
+    if fullversion:
+        suffix = "." + VERSION
+    else:
+        suffix = "." + VERSION.split(".")[0]
+
+    return select({
+        clean_dep("//tensorflow:macos_with_framework_shared_object"): [
+            clean_dep("//tensorflow:libtensorflow_framework%s.dylib" % suffix),
```

Comment:
@allenlavoie about the macos change here.

Reply:
Looks reasonable to me; sounds like name.major.minor.dylib is idiomatic.